tests/aws/services/apigateway/test_apigateway_basic.py

```diff
@@ -1978,7 +1978,9 @@ def test_api_gateway_sqs_integration_with_event_source(
         # create event source for sqs lambda processor
         event_source_data = {
             "FunctionName": integration_lambda,
-            "EventSourceArn": arns.sqs_queue_arn(sqs_queue),
+            "EventSourceArn": arns.sqs_queue_arn(
+                sqs_queue, account_id=TEST_AWS_ACCOUNT_ID, region_name=TEST_AWS_REGION_NAME
+            ),
```

Comment:
nit: I think we could use the fixture here for the sqs queue ARN.

Reply:
This is resolved on the merge from `master`